.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "perf/benchmark"]
```

Comment:
This seems not to be working. I tried all the submodule update and init command I have known but nothing happens. And I can't switch to Release build because
```
CMake Error at perf/CMakeLists.txt:24 (message):
  The submodules were not downloaded! GIT_SUBMODULE was turned off or failed.
  Please update submodules and try again.
```
But as I said, update does nothing. You might want to check this thing out: https://stackoverflow.com/questions/3336995/git-will-not-init-sync-update-new-submodules
It seems you somehow wrong added the submodule. I suppose you can reproduce it by doing a fresh clone in a new folder.

Reply:
Fixed! Sorry i add submodule to another branch, and then cherry-pick this commit< but cherry-pick was not work for submodule! Fixed!